website/content/en/status/report-2023-04-2023-06/puppet.adoc

```diff
@@ -0,0 +1,16 @@
+=== Puppet
+
+Links: +
+link:https://puppet.com/docs/puppet/latest/puppet_index.html[Puppet] URL: link:https://puppet.com/docs/puppet/latest/puppet_index.html[] +
+
+Contact: Puppet Team <puppet@FreeBSD.org>
+
+Puppet is a Free Software configuration management tool, composed of a source of trust (Puppet Server) that describes the expected configuration of machines with a domain-specific language, and an agent (Puppet Agent) on each node which enforce the actual configuration match the expected one. An optional database (PuppetDB) can be setup for reporting and describing advanced schemas where the configuration of a machine depends on the configuration of another one.
+
+Since our last status report, the Puppet Team maintained the Puppet 6 and Puppet 7 ports up to date.
+
+Puppet 8 has been recently released and was added to the ports tree.
+
+Puppet 6 has reached End of Life and has been deprecated.  It is now expired.  Users of Puppet 6 are therefore advised to update to Puppet 7 or Puppet 8.
+
+For now, the Puppet 7 remains the default Puppet version for ports depending on Puppet.  The Puppet Community is hard at work making sure the various Puppet modules work with the latest code and at the time of writing this report, updating to Puppet 8 may be challenging.  The situation is getting better every day, and we expect to switch Puppet 8 as the default version of Puppet in a few months, when the wave of modules update is finished.
```

Comment:
s/For now, the Puppet 7 remains the default/For now, Puppet 7 remains the default/

Puppet version for ports depending on Puppet.  The Puppet Community is hard at work making sure the various Puppet modules work with the latest code and at the time of writing this report, updating to Puppet 8 may be challenging.

s/The situation is getting better every day, and we expect to switch Puppet 8 as the default/The situation is getting better every day, and we expect to switch to Puppet 8 as the default/